clang/lib/DPCT/UserDefinedRules/PatternRewriter.cpp

```diff
@@ -66,6 +65,20 @@ static bool isRightDelimiter(char Character) {
   return Character == '}' || Character == ']' || Character == ')';
 }
 
+static bool getRightDelimiter(char Character) {
```

Comment:
```suggestion
static char getRightDelimiter(char Character) {
```

Reply:
That char to bool change for return type was a mistaken replacement when I committed

Changed it back properly now